v1/keys.go

```diff
@@ -16,20 +16,26 @@ package keyprotect
 
 import (
 	"context"
+	"time"
 )
 
+type GetInput struct {
+	Instance string `json:"-" location:"header" locationKey:"Bluemix-Instance"`
+	KeyID    string `json:"-" location:"url" locationKey:"KeyID"`
+}
+
 type WrapInput struct {
 	InstanceID string `json:"-" location:"header" locationKey:"Bluemix-Instance"`
 	KeyID      string `json:"-" location:"url" locationKey:"KeyID"`
 
-	Plaintext      []byte   `json:"plaintext,omitempty"`
+	Plaintext      *[]byte  `json:"plaintext,omitempty"`
 	AdditionalData []string `json:"aad,omitempty"`
 }
 
 type WrapOutput struct {
-	Plaintext  []byte                 `json:"plaintext"`
-	Ciphertext []byte                 `json:"ciphertext"`
-	KeyVersion map[string]interface{} `json:"keyVersion"`
+	Plaintext  *[]byte                `json:"plaintext, omitempty"`
+	Ciphertext *[]byte                `json:"ciphertext, omitempty"`
```

Comment:
Why did you switch these to pointers... I think that has a pretty big impact on how the serializer does its work...?

Reply:
I made this change while testing the wrap and unwrap. 
The response was returning plaintext as null value for wrap response.
To avoid the display of plain text in the response I switched them to pointers.